packages/notification-center/src/hooks/use-notifications.hook.ts

```diff
@@ -37,9 +37,13 @@ export function useNotifications(props?: IUseNotificationsProps) {
     await mapUpdateAction(messageId, actionButtonType, status, payload, storeId);
   }
 
-  async function refetch() {
+  const refetch = useCallback(async () => {
     await mapRefetch(storeId);
-  }
+  }, []);
+
+  useEffect(() => {
+    refetch();
+  }, [refetch]);
```

Comment:
A small question, why do we need this dependency here? Can't we just a call refetch on component mount?

Reply:
we don't need the dependency here, but it doesn't hurt to have it either